homeassistant/const.py

```diff
@@ -875,6 +875,7 @@ class UnitOfPrecipitationDepth(StrEnum):
 """Deprecated: please use UnitOfVolumetricFlux.INCHES_PER_HOUR"""
 
 # Concentration units
+CONCENTRATION_GRAMS_PER_CUBIC_METER: Final = "g/m³"
```

Comment:
Do we have any other integration in need of this unit?
If not, then maybe it should be added to local constants instead.

Reply:
Apparently Airq also uses it.